migrations/20180725202700-populate-collective-with-IRS-Bot.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const insert = (sequelize, table, entry) => {
+  return sequelize.query(`
+    INSERT INTO "${table}" ("${Object.keys(entry).join('","')}") VALUES (:${Object.keys(entry).join(",:")})
+  `, { replacements: entry });
+};
+
+module.exports = {
+  up: (queryInterface, sequelize) => {
+    
+    // Inserting OpenCollective Bot Collective
+    const botData = {
+      name: 'IRS Bot',
+      mission: 'Support users through the platform',
+      description: 'IRS bot that support users regarding IRS issues through the platform',
```

Comment:
```js
const botData = {
name: "W9 bot",
slug: "w9bot",
mission: "Help hosts by automating requesting users to submit their W9 or W8-BEN form when needed",
description: "Help hosts by automating requesting users to submit their W9 or W8-BEN form when needed",
longDescription: "Whenever someone files an expense to a host that has USD as its base currency, this bot will look at the sum of all past expenses of that user made during the year. If the sum exceeds $600, it will create a comment on the expense to ask to submit the W9 or W8-BEN form to the host",
...
}
```